src/OrchardCore.Modules/OrchardCore.Flows/Drivers/FlowPartDisplayDriver.cs

```diff
@@ -101,7 +101,7 @@ private IEnumerable<ContentTypeDefinition> GetContainedContentTypes(ContentTypeP
 
             return settings.ContainedContentTypes
                 .Select(contentType => _contentDefinitionManager.GetTypeDefinition(contentType))
-                .Where(t => t.GetSettings<ContentTypeSettings>().Stereotype == "Widget");
+                .Where(t => t != null && t.GetSettings<ContentTypeSettings>().Stereotype == "Widget");
```

Comment:
Is this case senstive or this should be lower case, coz I see the above query using `widget`

Reply:
Technically content type definitions, and stereotypes, are one of the few things that are case sensitive.
I'm not sure where you see `widget` in the query anywhere, it is always `Widget`

Reply:
Sorry, seems I was wrong when I was in my mobile, just leave my comment